src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala

```diff
@@ -139,7 +139,7 @@ object AppConfig {
     TermsOfServiceConfig(
       config.getBoolean("enabled"),
       config.getBoolean("isGracePeriodEnabled"),
-      config.getInt("version"),
+      config.getString("version"),
```

Comment:
I changed this to a string because I think we will want more descriptive version like a date or something